api/folders.js

```diff
@@ -0,0 +1,52 @@
+var fs = require('fs'),
+    path = require('path'),
+    mkdirp = require('mkdirp'),
+    _ = require('underscore'),
+    config = require('config'),
+    MQ = require('../message-queue');
+
+function Folders() {
+    this.base = './data';
+    
+    this.path = 'null';
```

Comment:
Why 'null' as a string and not a null object? Also, do we really want to deal with null checks?